cdc/sinkv2/ddlsink/mq/ddlproducer/kafka_ddl_producer.go

```diff
@@ -0,0 +1,182 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ddlproducer
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/Shopify/sarama"
+	"github.com/pingcap/errors"
+	"github.com/pingcap/log"
+	"github.com/pingcap/tiflow/cdc/contextutil"
+	"github.com/pingcap/tiflow/cdc/model"
+	"github.com/pingcap/tiflow/cdc/sink/mq/codec"
+	cerror "github.com/pingcap/tiflow/pkg/errors"
+	"go.uber.org/zap"
+)
+
+// Assert DDLEventSink implementation
```

Comment:
why we need these comments?

Reply:
Because there is no DDLEventSink before I rebase the PR.

Reply:
Please checkout [adeddfb](https://github.com/pingcap/tiflow/pull/6499/commits/adeddfb4d7879b6d9619b3cb9f29b8c45f0770e7)